### PR TITLE
Center-align "Back to..." link on attach pvc page

### LINKED ISSUE #2352: Storage empty state back link should be centered on dc add storage page?
current:
screen shot 2017-10-23 at 2 34 47 pm

proposed:
screen shot 2017-10-23 at 2 49 49 pm

### PR DESCRIPTION
**Pull request by rhamilto**
Fixes #2352 

screen shot 2017-10-23 at 4 29 17 pm

### CODE REVIEW

File: app/views/attach-pvc.html [hunk 1]
```diff
@@ -23,7 +23,7 @@ <h2 class="text-center">No persistent volume claims.</h2>
               To claim storage from a persistent volume, refer to the documentation on <a target="_blank" ng-href="{{'persistent_volumes' | helpLink}}">using persistent volumes</a>.
             </p>
```
Comment (spadgett) [inline app/views/attach-pvc.html]: Should we center this paragraph too for consistency with other empty state messages?

**Comment by rhamilto** [comment]
Moved .text-center to .empty-state-message so all nodes are centered.  Good catch, @spadgett.

**Pull request merged**